closed/OpenJ9.gmk

```diff
@@ -324,14 +325,22 @@ run-preprocessors-j9 : stage-j9
 			UMA_OPTIONS_EXTRA="-buildDate $(shell date +'%Y%m%d')" \
 			tools \
 	)
+ifneq (true,$(OPENJ9_ENABLE_CMAKE))
 	@$(SED) -i -e 's/^\s*VERSION_MAJOR\s*.*=.*/VERSION_MAJOR:=8/' $(OUTPUT_ROOT)/vm/makelib/mkconstants.mk
```

Comment:
I can't see that this is still required, considering mkconstants.mk doesn't assign VERSION_MAJOR to anything any more. I think this can just be deleted.

Reply:
@dnakamura if you remove this sed command completely I will PR test this and then merge if all goes well.